openquake/server/middleware.py

```diff
@@ -17,33 +17,33 @@
 # along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
 
 from django.conf import settings
-from django.core.urlresolvers import reverse
 from django.http import HttpResponseRedirect
 from re import compile
 
```

Comment:
` from re import compile` shadows a Python builtin called `compile`. Better to import `re` and then use `re.compile`